content/en/apps/guides/hosting/app-developer.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "App Developer Hosting"
+linkTitle: "App Developer Hosting"
+weight: 40
+description: >
+  Hosting the CHT when developing apps
+---
+
+{{% alert title="Note" %}} This guide assumes you are a CHT app developer wanting to either run concurrent instances of the CHT, or easily be able to switch between different instances without loosing any data while doing so. To do development on the CHT core itself, see the [development guide](https://github.com/medic/cht-core/blob/master/DEVELOPMENT.md). 
+
+To deploy the CHT in production, see either [AWS hosting]({{< relref "apps/guides/hosting/self-hosting.md" >}}) or [Self hosting]({{< relref "apps/guides/hosting/ec2-setup-guide.md" >}}){{% /alert %}}
+
+
+## Getting started
+
+Be sure to meet the [CHT hosting requirements]({{< relref "apps/guides/hosting/requirements" >}}) first. As well, if any other `medic-os` instances using [the main `docker-compose.yml` file](https://github.com/medic/cht-core/blob/master/docker-compose.yml) are running locally, stop them otherwise port, storage volume and container name conflicts may occur.
+
+After meeting these requirements, download the developer YAML file in the directory you want to store them:
+
+```shell script
+curl -o docker-compose-developer.yml https://raw.githubusercontent.com/medic/cht-core/master/docker-compose-developer-base.yml
+```
+
+To start the first developer CHT instance, run `docker-compose` and specify the file that was just download:
+
+```shell script
+docker-compose -f docker-compose-developer.yml up
+```
+
+This may take some minutes to fully start depending on the speed of the Internet connection and speed of the bare-metal host.  After it has completed, the CHT should be accessible on [https://localhost](https://localhost).
```

Comment:
Might explain what docker is doing that it might be taking long to pull the images. 

`This may take a while as docker is pulling the images and starting the application. This is based on the speed of the Internet connection and speed of the bare-metal host`

Reply:
Cool cool! I added a one liner in there per 1544ed0